roc/log_test.go

```diff
@@ -0,0 +1,100 @@
+package roc
+
+import (
+	"io/ioutil"
+	"log"
+	"testing"
+	"time"
+)
+
+const defaultLogLevel LogLevel = LogError
+
+type testWriter struct {
+	ch chan string
+}
+
+func makeTestWriter() testWriter {
+	return testWriter{
+		ch: make(chan string, 1),
```

Comment:
I think 1 is the default value, so you don't have to specify it here. But we can keep it like this too, this is not an error

Reply:
Nope, default value is 0: https://play.golang.org/p/YTtzluwniT6

Reply:
Also added a comment.

Reply:
I see, makes sense!